nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -557,6 +557,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - `make-disk-image` handles `contents` arguments that are directories better, fixing a bug where it used to put them in a subdirectory of the intended `target`.
 
+- The option `services.jitsi-videobridge.apis` has been renamed to `colibriRestApi` and turned into a boolean. Setting it to `true` will enable the private rest API, useful for monitoring using `services.prometheus.exporters.jitsi.enable`. Learn more about the API: "[The COLIBRI control interface (/colibri/)](https://github.com/jitsi/jitsi-videobridge/blob/9eefa65d62030e8733ed9c9cff999a8e310dd7ad/doc/rest.md)".
```

Comment:
Can we reference a tag here? What is that commit pointing to?

Reply:
Sure, just wanted to ensure the link does work in the future.